Directory.php

```diff
@@ -34,38 +34,13 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-namespace {
+namespace Hoa\File;
 
-from('Hoa')
-
-/**
- * \Hoa\File\Exception
- */
--> import('File.Exception.~')
-
-/**
- * \Hoa\File\Exception\FileDoesNotExist
- */
--> import('File.Exception.FileDoesNotExist')
-
-/**
- * \Hoa\File\Generic
- */
--> import('File.Generic')
-
-/**
- * \Hoa\File\Finder
- */
--> import('File.Finder')
-
-/**
- * \Hoa\Stream\Context
- */
--> import('Stream.Context');
-
-}
-
-namespace Hoa\File {
+use Hoa\File\Exception;
```

Comment:
What I was trying to say is: Remove every `from`/`import` and add `use` when necessary. “necessary” must be defined. If we are in the following namespace `Hoa\File`, then this is the top-namespace, so no need to declare `use Hoa\File\Exception` etc. We just need to declare other namespaces like `use Hoa\Stream`. However, if we are in `Hoa\File\Link` for instance and we use `Hoa\File\Generic`, we need to declare `use Hoa\File`.

Is it clearer?